src/frontend/context/BehandlingContext.ts

```diff
@@ -98,7 +98,7 @@ export const [BehandlingProvider, useBehandling] = constate(
         behandlingFakta,
         sluttDatoForrigeVedtak,
     }: Props): BehandlingContext => {
-        const { erSaksbehandler } = useApp();
+        const { erSaksbehandler, saksbehandler } = useApp();
```

Comment:
Mon tro om `saksbehandler` burde vært renamet til `innloggetSaksbehandler` for å minske potensiell forvirring